docker/grpc-exporter/app/server.py

```diff
@@ -0,0 +1,69 @@
+import os
+import time
+import subprocess
+import json
+from prometheus_client import start_http_server, Gauge
+
+
+class AppMetrics:
+    """
+    Representation of Prometheus metrics and loop to fetch and transform
+    application metrics into Prometheus metrics.
+    """
+
+    def __init__(self, polling_interval_seconds=60):
+        self.polling_interval_seconds = polling_interval_seconds
+
+        # Prometheus metrics to collect
+        self.ourMessages = Gauge("our_message_height", "Our Messages", ['peerId'])
+        self.theirMessages = Gauge("their_message_height", "Their Messages", ['peerId'])
+        self.divergenceSecondsAgo = Gauge("divergence_seconds_ago", "Divergence Seconds Ago", ['peerId'])
+        self.lastBadSync = Gauge("last_bad_sync", "Last Bad Sync", ['peerId'])
```

Comment:
I'm hesitant to be collecting this info for all peers without knowing how we'll be consuming it (do these metrics matter to us?)